src/components/CategoryFilter.js

```diff
@@ -0,0 +1,14 @@
+import PropTypes from 'prop-types';
+import { selectBox } from '../containers/BooksForm';
+
+const CategoryFilter = ({ filter }) => (
+  <select onChange={filter}>
+    {selectBox}
+  </select>
```

Comment:
- [ ] [OPTIONAL] it merges `All` select option` with available categories` [screenshot from Pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/milestone4/AllReturnsAllCategories.png)